IdentityCore/tests/MSIDPkeyAuthHelperTests.m

```diff
@@ -251,4 +294,59 @@ + (MSIDWPJKeyPairWithCert *)swizzled_getWPJKeysWithTenantId:(NSString *)tenantId
     return s_registrationInformationToReturn;
 }
 
```

Comment:
Added method to parse PKeyAuth response and check response for correct claim values.